docs/user_guides/debugging/stepping.rst

```diff
@@ -147,8 +147,8 @@ GDB output:
 ``set scheduler-locking step``
 -------------------------------
 
-Debug of the first line of the kernel and functions works out twice.
-This happens because you are debugging a multi-threaded program and multiple events may be received from different threads.
+The first line of the kernel and functions is debugged twice.
+This happens because you are debugging a multi-threaded program, so multiple events may be received from different threads.
```

Comment:
Aha! here you go. Maybe add the explanation everywhere or have one central know issues page.

Reply:
Agree, let's do it in a separate pr? And may be you could provide some example of "one central known issues page"?

Reply:
Ok.